src/main/java/htsjdk/samtools/util/BufferedLineReader.java

```diff
@@ -40,6 +41,27 @@
  * @author alecw@broadinstitute.org
  */
 public class BufferedLineReader extends LineNumberReader implements LineReader {
+    
+    private static float MAX_BYTES_PER_CHAR_UTF8 = Charset.forName("UTF8").newEncoder().maxBytesPerChar();
```

Comment:
Add `final`, use `StandardCharsets.UTF_8`:
```suggestion
    private static final float MAX_BYTES_PER_CHAR_UTF8 = StandardCharsets.UTF_8.newEncoder().maxBytesPerChar();
```